eth/protocol.go

```diff
@@ -17,17 +17,28 @@
 package eth
 
 import (
+	"fmt"
+	"io"
 	"math/big"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+// Constants to match up protocol versions and messages
+const (
+	eth61 = 61
+	eth62 = 62
+	eth63 = 63
+	eth64 = 64
```

Comment:
We might just as well compare against the numbers. Saves three characters each time.
`p.version >= 61` vs. `p.version >= eth61`.